common/beerocks/bwl/common/ap_wlan_hal.h

```diff
@@ -273,6 +316,13 @@ class ap_wlan_hal : public virtual base_wlan_hal {
      */
     virtual bool set_vap_enable(const std::string &iface_name, const bool enable) = 0;
     virtual bool get_vap_enable(const std::string &iface_name, bool &enable)      = 0;
+
+    /*!
+     * Generate "AP-STA-CONNECTED" events on connected clients on AP
+     * 
+     * @return true on success or false on error.
+     */
+    virtual bool generate_connected_clients_events() = 0;
```

Comment:
Why would you ever want to NOT get connected client events?

Reply:
It is used in passive mode when the ap_manager is transmitting and clients are able to connect it before the agent has established the link to the controller.